app/components/shared/event-item.tsx

```diff
@@ -1,16 +1,31 @@
 import { Link } from "@remix-run/react"
 
 import { ButtonLink } from "~/components/ui/button-link"
-import { formatPublished } from "~/utils/datetime"
+import {
+  formatPublished,
+  formatPublishedWithTime,
+  formatTime,
+} from "~/utils/datetime"
+import { Iconify } from "../ui/iconify"
 
 export interface Event {
   slug: string
   title: string
   description: string
   dateTimeStart: string
+  dateTimeEnd: string
   image: {
     url: string
   } | null
+  location: {
+    label: string
+  } | null
+  media: {
+    name: string
+  } | null
+  category: {
+    symbol: string
+  } | null
 }
```

Comment:
@fikrialwan Use the type from Prisma instead of manual

Reply:
I've use the type from Prisma for event item on this commit https://github.com/bandungdevcom/bandungdev.com/pull/42/commits/2f463a6f94bb11884fab867828a40eb6cd443ec9